### PR TITLE
add more datasets in kaggle

### PR DESCRIPTION
**Pull request by askender**

### CODE REVIEW

File: datasets.md [hunk 1]
```diff
@@ -77,3 +77,9 @@ If you would like to use a dataset that you don't see listed here, please submit
 | Cancer Cell Line Encyclopedia (CCLE) |  The Cancer Cell Line Encyclopedia (CCLE) project is an effort to conduct a detailed genetic characterization of a large panel of human cancer cell lines. This dataset contains RNA-Seq Aligned Reads, WXS Aligned Reads, and WGS Aligned Reads data. | 20.7 TiB |  Various | https://portals.broadinstitute.org/ccle/about |
 | CCAFS-Climate Data | High resolution climate data to help assess the impacts of climate change primarily on agriculture. These open access datasets of climate projections will help researchers make climate change impact assessments. | 6.8 TiB | Various | http://www.ccafs-climate.org/ |
 | ECMWF ERA5 Reanalysis |  ERA5 is the fifth generation of ECMWF atmospheric reanalyses of the global climate, and the first reanalysis produced as an operational service. | 7.6 Tib |  .nc | https://apps.ecmwf.int/datasets/ |
+| MusicNet Dataset | A curated collection of labeled classical music. | 31 GB | wav | https://www.kaggle.com/imsparsh/musicnet-dataset |
```
Comment (dkkapur) [inline datasets.md:80]: Looks good, useful open dataset with > 1k downloads.

File: datasets.md [hunk 2]
```diff
@@ -77,3 +77,9 @@ If you would like to use a dataset that you don't see listed here, please submit
...
+| CLEVR Dataset | A Diagnostic Dataset for Compositional Language and Elementary Visual Reasoning. | 19 GB | png&json | https://www.kaggle.com/timoboz/clevr-dataset |
```
Comment (dkkapur) [inline datasets.md:81]: Looks useful as well, lower downloads but might have leverage with > 5k views on Kaggle, I'm supportive of adding this as well

File: datasets.md [hunk 3]
```diff
@@ -77,3 +77,9 @@ If you would like to use a dataset that you don't see listed here, please submit
...
+| NSFW - not-safe-for-work | Classification dataset, This dataset contains content intended for individuals 18 or older. | 20 GB | jpg | https://www.kaggle.com/omeret/not-safe-for-work |
```
Comment (dkkapur) [inline datasets.md:82]: Relatively low impact, NSFW means content policy might be difficult to manage across regions etc. Let's not include this dataset for the time being

File: datasets.md [hunk 4]
```diff
@@ -77,3 +77,9 @@ If you would like to use a dataset that you don't see listed here, please submit
...
+| Magic kids dataset | A Machine Learning classification dataset. | 17 GB | jpg | https://www.kaggle.com/fedorshakhovskiy/magic-kids |
```
Comment (dkkapur) [inline datasets.md:83]: Relatively low impact - low views and downloads, suggest we skip this one for the time being as well.

File: datasets.md [hunk 5]
```diff
@@ -77,3 +77,9 @@ If you would like to use a dataset that you don't see listed here, please submit
...
+| COCO2017 | Common Objects in Context | 20 GB | jpg | https://www.kaggle.com/aishwr/coco2017 |
```
Comment (dkkapur) [inline datasets.md:84]: 3k+ views, 700+ downloads. Looks OK to include!

File: datasets.md [hunk 6]
```diff
@@ -77,3 +77,9 @@ If you would like to use a dataset that you don't see listed here, please submit
...
+| COCO 2017 Dataset | COCO is a large-scale object detection, segmentation, and captioning dataset. | 20 GB | jpg | https://www.kaggle.com/awsaf49/coco-2017-dataset |
```
Comment (dkkapur) [inline datasets.md:85]: Similar dataset source as previous? OK to include. This one is 26 GB though, table should reflect the correct size

**Review by dkkapur — CHANGES_REQUESTED** [review]
Thanks @askender for recommending these!
I am ok with including most of these but the NSFW and the Magic kids dataset one.

Changes requested:
1. remove these 2 datasets
2. fix size of last dataset - should be 26 GB

**Comment by askender** [comment]
I recreate a new one:  https://github.com/filecoin-project/slingshot/pull/453
Changes have been made:
- remove these 2 datasets
- fix size of last dataset - should be 26 GB

**Pull request closed without merging**